### PR TITLE
Feature/39 primary nav

### PR DESCRIPTION
**Pull request by chris-lawton**
Adds Primary Nav FE
Adds Search FE

### CODE REVIEW

File: rca/project_styleguide/templates/patterns/molecules/navigation/primarynav.html [hunk 1]
```diff
@@ -1,35 +1,122 @@
...
+    <div class="header__navigation-toggle">
+        {% include 'patterns/molecules/menu-toggle/menu-toggle.html' with modifier="header__menu-toggle--medium" %}
+        {% include 'patterns/molecules/search-toggle/search-toggle.html' %}
+
+        <a class="header__menu-close" href="#" data-close-menu>
+            <svg width="14" height="14"><use xlink:href="#close"></use></svg>
+        </a>
+    </div>
+
+    <span class="header__inner--menu-container">
```
Comment (nicklee) [inline rca/project_styleguide/templates/patterns/molecules/navigation/primarynav.html]: should we have `header__inner` here as well?

File: rca/project_styleguide/templates/patterns/molecules/navigation/standoutnav.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<ul class="nav nav--standout {{modifier}}">
+    <li>
+        <a class="nav__link nav__link--base link link--link" herf="#">
```
Comment (nicklee) [inline rca/project_styleguide/templates/patterns/molecules/navigation/standoutnav.html]: typo: href :)

File: rca/project_styleguide/templates/patterns/molecules/navigation/standoutnav.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            <span class="link__label">Contact us</span>
+            <svg width="12px" height="8px" class="link__icon"><use xlink:href="#arrow"></use></svg>
+        </a>
+    </li>
+    <li>
+        <a class="nav__link nav__link--base link link--link" herf="#">
```
Comment (nicklee) [inline rca/project_styleguide/templates/patterns/molecules/navigation/standoutnav.html]: same typo

File: rca/project_styleguide/templates/patterns/molecules/navigation/standoutnav.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+            <span class="link__label">Support Generation RCA</span>
+            <svg width="12px" height="8px" class="link__icon"><use xlink:href="#arrow"></use></svg>
+        </a>
+    </li>
+    <li>
+        <a class="nav__link nav__link--base nav__link--standout link link--link" herf="#">
```
Comment (nicklee) [inline rca/project_styleguide/templates/patterns/molecules/navigation/standoutnav.html]: same typo

File: rca/static_src/sass/components/_search-toggle.scss [hunk 5]
```diff
@@ -14,7 +14,7 @@
     }
 
     .app__navigation & {
-        margin-top: $grid * 5;
+        margin-top: $grid * 5.75;
```
Comment (nicklee) [inline rca/static_src/sass/components/_search-toggle.scss]: being pinickity on a friday afternoon, apologies ;) `($grid * 5.75)`

File: rca/project_styleguide/templates/patterns/molecules/navigation/primarynav.html (hunk 1 above)
Comment (chris-lawton) [reply rca/project_styleguide/templates/patterns/molecules/navigation/primarynav.html]: aye 👍

File: rca/project_styleguide/templates/patterns/molecules/navigation/standoutnav.html (hunk 4 above)
Comment (chris-lawton) [reply rca/project_styleguide/templates/patterns/molecules/navigation/standoutnav.html]: can you tell i copy and pasted

File: rca/static_src/sass/components/_search-toggle.scss (hunk 5 above)
Comment (chris-lawton) [reply rca/static_src/sass/components/_search-toggle.scss]: i'll let you off this time

**Pull request merged**